helpers/checkpoint_logger/prometheus.py

```diff
@@ -67,28 +126,46 @@ class PrometheusCheckpointLoggerHandler:
     methods in this class are mainly used by the CheckpointLogger class.
     """
 
-    def log_begun(self, flow: str):
+    def log_begun(self, flow: str, repo_id: int = None):
```

Comment:
```suggestion
    def log_begun(self, flow: str, repo_id: int | None = None):
```
same here, the argument type needs to match up with the default value.